lib/sidebar.js

```diff
@@ -21,12 +21,10 @@ define(function () {
     sidebar.appendChild(container);
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || sidebar.firstElementChild.offsetHeight === 0) {
```

Comment:
`||` is OK, if sidebar is closed (hidden) or is sidebar is on bottom 0 is correct value. If sidebar is visible we need the offset.

you are right about the `button`, the variable is available. But no more static width in code :).